tests/components/media_player/test_async_helpers.py

```diff
@@ -158,6 +158,19 @@ async def test_volume_down(
     assert player.volume_level == 0.5 - volume_step
 
 
+async def test_volume_set(hass: HomeAssistant) -> None:
+    """Test the volume_set method."""
+    player = SimpleMediaPlayer(hass)
+    assert player.volume_level == 0
+    await player._async_set_volume_level(0.5, 0.01)
```

Comment:
I don't like that we're calling a private method here, but I perhaps it's OK since we're testing the helpers directly vs. calling them through a service action.

Reply:
Yea i didn't either but couldn't see an alternative that made sense.